Assets/_Project/Scripts/UserInterface/Popups/PopupPayload.cs

```diff
@@ -0,0 +1,16 @@
+﻿using UnityEngine;
+
+namespace PolSl.UrbanHealthPath.UserInterface.Popups
+{
+    public class PopupPayload
```

Comment:
Payload sounds a bit scary in this case xd

Reply:
Not as scary as 2.0 z pbla.